### ISSUE TITLE
Override/delete a planned absence

### ISSUE THREAD

**Issue by tbar0970** [tbar0970/jethro-pmm#951]
Joe Bloggs told me he's unavailable 17 Sept so I saved a planned absence.
But now things have changed, he's available, and I want to put him on the roster for 17 Sept.
So I go to the roster, but can't select him because of the planned absence.
It's clunky to have to find his person record, delete the planned absence, then come back to the roster and add him.

It'd be good if, when the edit-roster page tells me Joe Bloggs is unavailable, there's some way to click through to manage his planned absences.  Not sure exactly how that could be implemented UI-wise.

**Comment by tim-pearce** [comment]
It may have been 'fixed' in the current version of Jethro but in the version I am running I can search for a person who is absent and just add them to the roster. The only difference is that the person is flagged on screen as absent.
I used this 'feature' last Sunday for myself. I was not at the service so marked as absent, however I was available after the service to do the banking.

**Comment by jefft** [comment]
> It'd be good if, when the edit-roster page tells me Joe Bloggs is unavailable, there's some way to click through to manage his planned absences. Not sure exactly how that could be implemented UI-wise.

Like a 'Planned Absences' pseudo-role? See #911

**Comment by tbar0970** [comment]
Yeah that'd be one solution, if the listed names were links. But something that takes up less space would be preferable.  Or some way to click on the disabled option and say "I don't care, put them on anyway and delete the planned absence"

**Comment by tbar0970** [comment]
> t may have been 'fixed' in the current version of Jethro but in the version I am running I can search for a person who is absent and just add them to the roster. The only difference is that the person is flagged on screen as absent.
> I used this 'feature' last Sunday for myself. I was not at the service so marked as absent, however I was available after the service to do the banking.

Yeah that's been fixed in the latest version.
